src/Avalonia.Base/Media/Fonts/FontCollectionKey.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Avalonia.Media.Fonts
+{
+    public readonly struct FontCollectionKey
```

Comment:
Perfect candidate for "public readonly record struct FontCollectionKey()"
https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoBBmABM3AYVwG9NcL88EAWXAWQAoBKU8ygX0y40x1wBOAUwgATAPYwANgE9BQgMbiBo3AGcALgICuCjbgBikjYXFSpijVEkBpITMZGYGgMoaZF3G49CANIeMAdSEoAHMAC31gsMj/J1ctIQ0FcK9E5PDmAG5MPjwhGG0wAOdvCxIefgKiko1oiI0KvNxq4vi3YQyKoA==